src/game/SpellEffects.cpp

```diff
@@ -1149,6 +1149,13 @@ void Spell::EffectDummy(SpellEffectIndex eff_idx)
 
                     return;
                 }
+                case 32027:                                 // Expedition Flare
+                {
+                    if (!m_CastItem)                        // 32029 = Expedition Preserver | 32030 = Expedition Scout
```

Comment:
remove ! as we now use the if condition for the reverse functionality =)
Also I would move the comment on this line back a few spaces; in my opinion they shouldn't be alligned with the case comments but it's not a biggie.